client/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="images/favicon.png" />
```

Comment:
This shouldn't work. 
favicon isn't added to the assets yet. If you want to add a logo u can use /images/logo.png for now

Reply:
I made a copy of the logo as a favicon.png so that it won't get messed up